### DISCUSSION TITLE
raspberry pi 5 bookworm camera v3 not found

### DISCUSSION THREAD

**Discussion by suosuo6** [bluenviron/mediamtx#2754] [Questions]
### Question

I have tried both CSI ports
docker run --rm -it --network=host --privileged --tmpfs /dev/shm:exec -v /run/udev:/run/udev:ro -e MTX_PATHS_CAM_SOURCE=rpiCamera bluenviron/mediamtx:latest-rpi
2023/11/28 13:39:58 INF MediaMTX v1.3.1
2023/11/28 13:39:58 INF configuration loaded from /mediamtx.yml
2023/11/28 13:39:58 INF [path cam] [RPI Camera source] started
2023/11/28 13:39:58 INF [RTSP] listener opened on :8554 (TCP), :8000 (UDP/RTP), :8001 (UDP/RTCP)
2023/11/28 13:39:58 INF [RTMP] listener opened on :1935
2023/11/28 13:39:58 INF [HLS] listener opened on :8888
2023/11/28 13:39:58 INF [WebRTC] listener opened on :8889 (HTTP), :8189 (ICE/UDP)
2023/11/28 13:39:58 INF [SRT] listener opened on :8890 (UDP)
[0:32:34.424405642] [13] INFO Camera camera_manager.cpp:297 libcamera v0.0.5+83-bde9b04f
2023/11/28 13:39:59 ERR [path cam] [RPI Camera source] camera_create(): selected camera is not available
[0:32:39.569273539] [19] INFO Camera camera_manager.cpp:297 libcamera v0.0.5+83-bde9b04f
2023/11/28 13:40:04 ERR [path cam] [RPI Camera source] camera_create(): selected camera is not available

**Comment by aler9** [comment]
Hello, in Raspberry Pi 5, the rpicamera source can't be used anyway because the device doesn't have a H264 hardware encoder anymore. Please see here: #2581

[a comment by an automated account is not shown]
